mixs-templates/README.md

```diff
@@ -1,11 +1,17 @@
 ## MIxS (meta)data collection spreadsheet templates
 
-This folder contains the MIxS schema (meta)data collection templates in the Excel spreadsheet (.xlsx) format. Each of the templates in this folder are blank spreadsheet templates with only the header row populated with the names of the terms that are associated with a particular [checklist](https://sujaypatil96.github.io/mixs/#checklists), [extension](https://sujaypatil96.github.io/mixs/#extensions), or [combination](https://sujaypatil96.github.io/mixs/combinations/) (of one checklist plus one extension). The templates offer pulldown menus for filling out columns for which there is a brief [enumeration](https://sujaypatil96.github.io/mixs/enumerations/) of permissible values, like [window_type](https://sujaypatil96.github.io/mixs/0000856/), but this [feature is disabled](https://linkml.io/linkml/generators/excel.html) if the total character length of the permissible values is more than 255, like [fao_class](https://sujaypatil96.github.io/mixs/0001083/).
+In this folder, the GSC provides Microsoft Excel (.xlsx) spreadsheets that can be used as templates for collecting metadata according to the MIxS standard. The GSC appreciates that spreadsheets like these can be very useful for preparing data to be submitted to an INSDC database. Users should be aware of the limitations of these spreadsheets, listed below.
 
-Note: The [linkml-validate](https://linkml.io/linkml/data/validating-data.html#the-linkml-validate-cli) command-line tool can be used to check the validity of a completed template, but no validation is provided in the template itself.
+Each of the files in this folder and its subfolders are blank template spreadsheets, with only the header row populated with the names of terms associated with a particular [Checklist](https://sujaypatil96.github.io/mixs/#Checklists), [Extension](https://sujaypatil96.github.io/mixs/#Extensions), or [Combination](https://sujaypatil96.github.io/mixs/combinations/) (of one Checklist plus one Extension). 
```

Comment:
The links here point to @sujaypatil96 's page - this should point to a GSC official site/repo

Reply:
thanks